pkg/discovery/probe/node_probe.go

```diff
@@ -196,11 +208,26 @@ func (nodeProbe *NodeProbe) createCommoditySold(node *api.Node, nodePodsMap map[
 		}
 	}
 
-	// Use Kubernetes service UID as the key for cluster commodity
+	// Create a new VMPMAccessCommodity to indicate if the node is schedulable. If the Node is schedulable, then
+	// the node will sell it. Otherwise, it doesn't sell. On the consumer side, if a pod can be scheduled by
```

Comment:
"if a pod can be scheduled by scheduler, then the pod should buy the same VMPMAccessCommodity". So if a pod cannot be scheduled by scheduler, it won't by this VMPMAccessCommodity? But not buying an accesscommodity will not provide any constraint, right? In that case, do we still want to decide which pods should buy this commodity, and which ones shouldn't?